src/core/decode_stream.js

```diff
@@ -32,13 +32,29 @@ class DecodeStream extends BaseStream {
     this.bufferLength = 0;
     this.eof = false;
     this.buffer = emptyBuffer;
-    this.minBufferLength = 512;
-    if (maybeMinBufferLength) {
-      // Compute the first power of two that is as big as maybeMinBufferLength.
-      while (this.minBufferLength < maybeMinBufferLength) {
-        this.minBufferLength *= 2;
-      }
+    this.minBufferLength = DecodeStream._makeBigEnough(
+      512,
+      maybeMinBufferLength
```

Comment:
Given that `maybeMinBufferLength` may be undefined/null here, will the new helper method work correctly in that case?